From fdeb66cd4dd8ba4e0b19e1c5e568b0f86ddf2912 Mon Nov 11 22:51:52 2019
From: Raphael Kubo da Costa <raphael.kubo.da.costa@intel.com>
Date: Mon Nov 11 22:51:52 2019
Subject: [PATCH] [bluetooth]: Include <vector> for std::vector

IWYU. This fixes the libstdc++ build after commit df930962a2 ("De-duplicate
multiple implementations of bluetooth address and UUID parsing"):

../../device/bluetooth/public/cpp/bluetooth_uuid.h:87:8: error: no template named 'vector' in namespace 'std'
  std::vector<uint8_t> GetBytes() const;
  ~~~~~^

Bug: 957519
Change-Id: I023a57e87cfac39cac2d1db0079458a562391424
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/1910082
Auto-Submit: Raphael Kubo da Costa <raphael.kubo.da.costa@intel.com>
Commit-Queue: Giovanni Ortuño Urquidi <ortuno@chromium.org>
Reviewed-by: Giovanni Ortuño Urquidi <ortuno@chromium.org>
Cr-Commit-Position: refs/heads/master@{#714297}
---

diff --git a/device/bluetooth/public/cpp/bluetooth_uuid.h b/device/bluetooth/public/cpp/bluetooth_uuid.h
index e3d5b44..0c27ae7 100644
--- a/device/bluetooth/public/cpp/bluetooth_uuid.h
+++ b/device/bluetooth/public/cpp/bluetooth_uuid.h
@@ -6,6 +6,7 @@
 #define DEVICE_BLUETOOTH_PUBLIC_CPP_BLUETOOTH_UUID_H_
 
 #include <string>
+#include <vector>
 
 #include "build/build_config.h"
